docs/source/automodules/client.rst

```diff
@@ -3,4 +3,8 @@ Client
 
 .. automodule:: cellarium.cas.client
    :members:
-   :show-inheritance:
\ No newline at end of file
+
+.. autoclass:: cellarium.cas.constants.CountMatrixInput
```

Comment:
Note that now I'm just pulling in classes and not the entire modules.  Let's us have better control over what makes it into the docs